app.js

```diff
@@ -16,18 +17,21 @@ app.get("/api/articles/:article_id", getArticleById);
 app.patch("/api/articles/:article_id", patchArticleById);
 app.get("/api/users", getUsers);
 app.get("/api/articles", getArticles);
-app.get("/api/articles/:article_id/comments", getAllCommentsByArticleId)
+app.get("/api/articles/:article_id/comments", getAllCommentsByArticleId);
+app.post("/api/articles/:article_id/comments", postCommentByArticleId)
 
 app.all("/*", (req, res) => {
   res.status(404).send({ msg: "Not Found!" });
 });
 
 app.use((err, req, res, next) => {
-  if (err.code === "22P02") {
+  if (err.code === "22P02" || err.code === "23503") {
     res.status(400).send({ msg: "Invalid input data type" });
   } else if (err.code === "23502") {
     res.status(400).send({ msg: "Missing required fields." });
-  } else if (err.status) {
+  // } else if (err.code === "23503") {
+  //   res.status(404).send({msg: "Article not found!"});
+} else if (err.status) {
```

Comment:
Why is this code commented out?

Reply:
Forgot to delete that, it's handled above